mathics/core/definitions.py

```diff
@@ -1066,6 +1065,10 @@ def load_builtin_definitions(
     """
     Load definitions from Builtin classes, autoload files and extension modules.
     """
+    from mathics.core.load_builtin import (
```

Comment:
I ran into circular dependencies from changes in #1413, and this fixed that. 

However, moving this down _also_ feels right in that there are the other imports here. 

What this is telling us is that `load_buildin_definitions()` should appear in some _other_ module.